### DISCUSSION TITLE
NEOGEO Pocket Color Selection vol.1

### DISCUSSION THREAD

**Discussion by Vesuvius72** [farmerbb/RED-Project#30] [General]
I hope someone can figure out how to extract these Neo Geo  Pocket Color roms from this collection. These have been on Switch for awhile, but SNK just brought this collection to PC via Steam. It would be great to run the original roms in Retroarch. On a separate note, if your wanting SNK VS. CAPCOM: THE MATCH OF THE MILLENNIUM then you will have to purchase it separately from this collection.

**Comment by farmerbb** [comment]
I just bought the standalone SNK vs. Capcom and it turns out that the ROM and BIOS are just located in a folder.  Assuming the rest of the collection uses the same engine, should be the same situation there.

![Screenshot 2021-10-01 10 48 29 AM](https://user-images.githubusercontent.com/6822476/135658056-555e1596-ac17-4b60-959f-0d8a46719a33.png)

**Reply by Vesuvius72** [reply]
Nice! I will buy the full collection tonight along with this game. Let's hope the full collection is set up the same way.

**Comment by farmerbb** [comment]
Went ahead and bought the rest of the NGPC Selection, and yes, the ROMs are just loose files located in the game data here as well.

![Screenshot 2021-10-04 9 32 26 AM](https://user-images.githubusercontent.com/6822476/135880474-b9aa1cd4-cbf1-4301-a346-5a1d0f5f429a.png)
